### PR TITLE
Build/schema

### PR DESCRIPTION
**Pull request by HyflerSD**
Draft schema structure for database tables

### CODE REVIEW

File: database/migrations/2024_02_09_035707_create_students_table.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up()
+    {
+        Schema::create('students', function (Blueprint $table) {
+            $table->id('StudentID');
+            $table->string('Name', 100);
```
Comment (Rolindets) [inline database/migrations/2024_02_09_035707_create_students_table.php]: Should we have a first name and last name column? easier to query if we ever need it?

**Review by Rolindets — COMMENTED** [review]
Left a comment for name of student. Let me know what you think before i approve/merge

**Review by Rolindets — APPROVED** [review]
LGTM

File: database/migrations/2024_02_09_035707_create_students_table.php (hunk 1 above)
Comment (patryhernandez0724) [reply database/migrations/2024_02_09_035707_create_students_table.php]: if we're only using one column for people's names, it's probably a good idea to call it "Full Name" so it includes both their first and last names. Having two columns should be a good idea to make it clearer and more organized.

**Pull request merged**